LICENSE.txt

```diff
@@ -9,7 +9,7 @@ software.
 
 1. Definitions
 
-The terms "reproduce," "reproduction," "derivative works," and "distribution"
+The terms "reproduce", "reproduction", "derivative works" and "distribution"
```

Comment:
Yeah...  thanks for looking out for this @PAVBAN95, but this is correct both grammatically and to the original license.